mcu/atmega-hal/src/adc.rs

```diff
@@ -1,56 +1,8 @@
 //! Analog-to-Digital Converter
 
-use crate::port;
+use crate::{pac, port::*};
```

Comment:
Not a fan of the wildcard import here as it is impossible to see why the import is needed.  The previous version made this explicit by naming the pins as `port::###`.